apps/web/public/static/locales/en/common.json

```diff
@@ -1390,11 +1384,13 @@
   "code_copied": "Code copied!",
   "how_you_want_add_cal_site": "How do you want to add {{appName}} to your site?",
   "choose_ways_put_cal_site": "Choose one of the following ways to put {{appName}} on your site.",
+  "setting_up_zapier": "Setting up your Zapier integration",
   "setting_up_make": "Setting up your Make integration",
   "generate_api_key": "Generate API key",
   "generate_api_key_description": "Generate an API key to use with {{appName}} at",
   "your_unique_api_key": "Your unique API key",
   "copy_safe_api_key": "Copy this API key and save it somewhere safe. If you lose this key you have to generate a new one.",
+  "zapier_setup_instructions": "<0>Log into your Zapier account and create a new Zap.</0><1>Select Cal.com as your Trigger app. Also choose a Trigger event.</1><2>Choose your account and then enter your Unique API Key.</2><3>Test your Trigger.</3><4>You're set!</4>",
```

Comment:
Why are these zapier strings added? If I am not mistaken the Zapier setup screen was removed